### ISSUE TITLE
No module named 'tensorflow.python.keras.applications.ResNet50'

### ISSUE THREAD

**Issue by UHayyat007** [tensorflow/tensorflow#35769]

**Comment by UHayyat007** [comment]
![image](https://user-images.githubusercontent.com/56769420/72201840-099a8f00-347a-11ea-9d44-5b5820c9731d.png)
i have this issue anyone here who can help???

**Comment by sagnik106** [comment]
Try import tensorflow.keras.applications.ResNet50

**Comment by oanush** [comment]
@UHayyat007 ,
Hi, please try using 
`from keras_applications.resnet import ResNet50`

Or if you just want to use ResNet50

`from keras.applications.resnet50 import ResNet50`
for more info please refer the [link](https://stackoverflow.com/questions/54682539/i-am-not-able-to-import-resnet-from-keras-applications-module).

**Comment by UHayyat007** [comment]
thank you

**Comment by oanush** [comment]
Closing since the issue is resolved.Thanks!

**Comment by soaring52** [comment]
This works for me with 
tensorflow ver.: 2.0.0
keras ver.: 2.2.4-tf
python ver.: 3.7.6 (default, Jan  8 2020, 20:23:39) [MSC v.1916 64 bit (AMD64)]

**from tensorflow.python.keras.applications import ResNet50**
ResNet50_model_ = ResNet50(weights='imagenet')

**Comment by Saxingh** [comment]
Hi,
I'm having a very similar issue with not being able to import the ResNet50 module with the module not being found.
I have tried both:
`import tensorflow.python.keras.applications.ResNet50
from keras.applications.resnet50 import ResNet50`
as well as, from the original code (it's not mine)
`from keras.applications.resnet50 import ResNet50`
I've tried combinations of others but cant seem to get any solution to work.

Im running the code in a miniconda env
pip version: 21.2.4
python 3.9.11
keras ver.: 2.8.0
tensorflow ver.: 2.8.0
I also have installed visual studio 2015, '17 and '19, CUDA installed on compatible GPU and checked %PATH% to ensure cuDNN64_8.dll is enabled.

Thanks!
